src/openfhe.jl

```diff
@@ -115,8 +115,8 @@ end
 function bootstrap!(secure_vector::SecureVector{<:OpenFHEBackend})
```

Comment:
```suggestion
function bootstrap(secure_vector::SecureVector{<:OpenFHEBackend})
```
This should now be called `bootstrap`, since it does not modify its argument anymore.

Since removing `bootstrap!` would be a breaking change, I recommend to just formally deprecate it using `Base.@deprecate` (see [this blog post](https://invenia.github.io/blog/2022/06/17/deprecating-in-julia/) for a good overview over handling deprecations in Julia)

Reply:
Oh, but I just realized: Instead of deprecating it, what about just _modifying_ the `secure_vector.data` entry? That is, while it is true that the OpenFHE.jl `Ciphertext` will change, we might be able to reuse the surrounding `SecureVector`?

Reply:
Done, It's a good idea, but for this purpose I had to declared `SecureVector` as mutable.

Reply:
If it's immutable (the default), there are many more optimizations a compiler might perform. For example, they can be created on the stack (thus no dynamic memory allocation is required).